coffea/processor/executor.py

```diff
@@ -1042,7 +1070,7 @@ def run_uproot_job(fileset,
     savemetrics = executor_args.pop('savemetrics', False)
     flatten = executor_args.pop('flatten', False)
     mmap = executor_args.pop('mmap', False)
-    nano = executor_args.pop('nano', False)
```

Comment:
To help with migration, can we leave `nano` here, and issue a deprecation warning? e.g.
```python
if nano:
    warnings.warn("Please use 'schema': coffea.nanoaod.NanoEvents rather than 'nano': True to enable awkward0 NanoEvents processing", DeprecationWarning)
    schema = NanoEvents
```